developwithpurpose/front-end-developer-exercise
# Hunter Metts Exercise

# Description

This pull request was made for an exercise in association with an open QA position at Ramsey Solutions.

When opened the user should be able to click through the 7 different baby steps and see a basic description of the baby step as well as what friends are on which step.

In addition to functionality the user should clearly see an animation when they switch between baby steps and that the CSS matches the mockup provided.

# Testing Instructions

```git fetch --all```
```git checkout HunterMettsExercise```
Click through and see that this project follows the criteria as stated in the readme of this repo

# Checklist:

- [X] I have commented my code and the process in the readme provided
